src/server/system_services/system_server.js

```diff
@@ -339,6 +339,8 @@ async function create_system(req) {
     try {
         const account_id = system_store.new_system_store_id();
         const changes = new_system_changes(name, account_id);
+        bucket_server.auto_setup_tier2(req, changes, true);
```

Comment:
I think it would make it a bit more robust if this call was inside new_system_changes...